package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.7",
   "description": "A simple router for MobX",
   "main": "dist/mobx-router.js",
-  "jsnext:main":"dist/mobx-router.es2015.js",
+  "jsnext:main": "dist/mobx-router.es2015.js",
```

Comment:
Would be nice to also include "module": "dist/mobx-router.es2015.js", as that has become the standard format